pygmt/src/tilemap.py

```diff
@@ -103,6 +113,13 @@ def tilemap(
         will stop trying to fetch more tiles from a rate-limited API [Default
         is ``2``].
 
+    zoom_adjust
+        The amount to adjust a chosen zoom level if it is chosen automatically. Values
+        outside of -1 to 1 are not recommended as they can lead to slow execution.
+
+        .. note::
+           The `zoom_adjust` parameter requires ``contextily>=1.5.0``.
```

Comment:
Also need to update here:
```suggestion
           The ``zoom_adjust`` parameter requires ``contextily>=1.5.0``.
```